src/main/java/brooklyn/entity/proxy/aws/ElbControllerImpl.java

```diff
@@ -58,6 +58,9 @@
 
     private static final Logger LOG = LoggerFactory.getLogger(ElbControllerImpl.class);
 
+    private ELBApi elbApi;
+    private LoadBalancerServiceContext loadBalancerServiceContext;
```

Comment:
ok, that makes sense. it seems safer to use them both as local variables, so ill make that change